replicator/Makefile

```diff
@@ -15,6 +15,7 @@ build:
 	@echo ${CACHE} >> .Dockerfile
 	@cat Dockerfile >> .Dockerfile
 	@echo "RUN rm -f /etc/apt/apt.conf.d/01proxy" >> .Dockerfile
+	@cat .singularity >> .Dockerfile
```

Comment:
You can add your Dockerfile enhancements directly into the Dockerfile.  

Is there another reason to keep it in .singularity?

Reply:
Just trying to be minimally invasive.

I'll integrate the changes directly into the Dockerfile.